benchmarks/templates/benchmarks/tutorials/models/deepdive_1.html

```diff
@@ -130,155 +112,148 @@ <h3 class="benefits_heading is-size-3-mobile">Part 4: Exploring the  __init__.py
+1       torchvision
+2       torch
+    </code></pre>
+    <p class="benefits_info is-size-5-mobile shorter has-text-weight-bold is-italic">
+        Note: The <span class="special_format">requirements.txt</span> file is optional because a
+        <span class="special_format">pyproject</span> file can also be used to add requirements.
+        If your model needs no extra packages, you can exclude this file from the submission package entirely.
     </p>
 </div>
 <div class="box leaderboard-table-component">
     <h3 class="benefits_heading is-size-3-mobile">Part 8: Putting it All Together </h3>
+
+
     <p class="benefits_info is-size-5-mobile shorter">
-        You are almost done! If you were actually submitting a model, the final step would be to run your model locally,
-        to ensure that everything is in working order. You can do this by running the
-        <span class="special_format">model.py</span> file itself. Please note that this can take ~5-10 minutes on a 2023
-        M1 Max MacBook Pro, so your run times may vary.
-        Once ran, this should produce the message below, indicating that you are ready for submission:
-        <pre class="modified_1"><code>
-Test successful, you are ready to submit!
+        You are almost done! If you were actually submitting a model, the final step prior to submission would be to run
+        your model locally to ensure that everything is in working order. You can do this by first following the
+        <a href="https://www.brain-score.org/tutorials/models/quickstart">Quickstart Tutorial</a> to install the
+        necessary packages for Brain-Score. Then, (as explained in the Quickstart) make sure your working directory is
+        <span class="special_format">/vision</span>, and run the command below to score the model
+        <span class="special_format">YOUR_MODEL_IDENTIFIER</span> on the publicly available data of a benchmark
+        called <span class="special_format">MajajHong2015public.IT-pls</span>.  (NOTE: Make sure to replace YOUR_MODEL_IDENTIFIER
+        with the actual model identifier you used to add your mode to the Brain-Score model registry in __init__.py.)
+
+    <pre class="modified_1"><code >
+python brainscore_vision score --model_identifier='YOUR_MODEL_IDENTIFIER' --benchmark_identifier='MajajHong2015public.IT-pls'
     </code></pre>
 
+    <p class="benefits_info is-size-5-mobile shorter">
+        Upon scoring completion, you should get a message like below:
+    </p>
+    <pre class="modified_1"><code >
+&lt;xarray.Score ()&gt;
+array(0.07637264)
+Attributes:
+    error:                 &lt;xarray.Score ()&gt;\narray(0.00548197)
+    raw:                   &lt;xarray.Score ()&gt;\narray(0.22545106)\nAttributes:\...
+    ceiling:               &lt;xarray.DataArray ()&gt;\narray(0.81579938)\nAttribut...
+    model_identifier:      pixels
+    benchmark_identifier:  MajajHong2015public.IT-pls
+    comment:               layers: {'IT': 'pixels'}
```

Comment:
I'm wondering if it would be helpful to use the actual `Resnet50_tutorial` output on Majajhong. That way, under `Attributes: comment:` it doesn't say `layers: {'IT': 'pixels'}` which might be a bit confusing?

Reply:
good catch.  I suggest that using YOUR_MODEL_IDENTIFIER here would be even less confusing given where this falls in the tutorial (talking about their submission package at that point).  I have made my change, but will change it to resnet50 if you prefer.